### ISSUE TITLE
Elvui character pave bugs out

### ISSUE THREAD

**Issue by ZebGasm** [jaredbwasserman/drift#46]
https://i.imgur.com/bLd2S7B.png
it bug out and becomes un usable

**Comment by jaredbwasserman** [comment]
I am unable to reproduce this issue. What version of ElvUI and Drift are you using? Also, which version of WoW is this (retail or classic)?

**Comment by jaredbwasserman** [comment]
Nevermind, I can repro this in retail. Looking into it.

**Comment by jaredbwasserman** [comment]
Looks like this is only reproducible the first time the frame is opened after a fresh login. Following a UI reload, issue no longer occurs.

```
3x Action[SetPoint] failed because[SetPoint would result in anchor family connection]: attempted from: CharacterFrame:SetPoint.
[string "=[C]"]: ?
[string "=[C]"]: in function `SetPoint'
[string "@FrameXML/UIParent.lua"]:2989: in function `UpdateUIPanelPositions'
[string "@FrameXML/UIParent.lua"]:2605: in function <FrameXML/UIParent.lua:2594>
[string "=[C]"]: in function `SetAttribute'
[string "@FrameXML/UIParent.lua"]:3435: in function <FrameXML/UIParent.lua:3433>
[string "=[C]"]: in function `securecall'
[string "@FrameXML/UIParent.lua"]:2886: in function `SetUIPanel'
[string "@FrameXML/UIParent.lua"]:2733: in function `ShowUIPanel'
[string "@FrameXML/UIParent.lua"]:2598: in function <FrameXML/UIParent.lua:2594>
[string "=[C]"]: in function `SetAttribute'
[string "@FrameXML/UIParent.lua"]:3196: in function `ShowUIPanel'
[string "@FrameXML/CharacterFrame.lua"]:21: in function `ToggleCharacter'
[string "TOGGLECHARACTER0"]:1: in function <[string "TOGGLECHARACTER0"]:1>

Locals:
(*temporary) = CharacterFrame {
 BottomLeftCorner = Texture {
 }
 Center = Texture {
 }
 RightEdge = Texture {
 }
 LeftEdge = Texture {
 }
 0 = <userdata>
 OnBackdropLoaded = <function> defined @SharedXML/Backdrop.lua:152
 DriftModifiable = true
 SetPortraitShown = <function> defined @SharedXML/PortraitFrame.lua:91
 GetBackdropColor = <function> defined @SharedXML/Backdrop.lua:390
 BottomEdge = Texture {
 }
 TopEdge = Texture {
 }
 GetTitleText = <function> defined @SharedXML/PortraitFrame.lua:28
 SetBackdropBorderColor = <function> defined @SharedXML/Backdrop.lua:422
 SetPortraitTexCoord = <function> defined @SharedXML/PortraitFrame.lua:87
 BottomRightCorner = Texture {
 }
 SetPortraitTextureRaw = <function> defined @SharedXML/PortraitFrame.lua:79
 CharacterInfoHooked = true
 template = "Transparent"
 TopRightCorner = Texture {
 }
 DriftHookSet = true
 ApplyBackdrop = <function> defined @SharedXML/Backdrop.lua:294
 TitleContainer = Frame {
 }
 ClearBackdrop = <function> defined @SharedXML/Backdrop.lua:282
 SetTitleColor = <function> defined @SharedXML/PortraitFrame.lua:32
 GetBackdropBorderColor = <function> defined @SharedXML/Backdrop.lua:409
 Tabs = <table> {
 }
 TopLeftCorner = Texture {
 }
 GetBackdrop = <function> defined @SharedXML/Backdrop.lua:347
 ItemLevelText = FontString {
 }
 SetTitle = <function> defined @SharedXML/PortraitFrame.lua:36
 Inset = CharacterFrameInset {
 }
 SetBorderBlendMode = <function> defined @SharedXML/Backdrop.lua:266
 backdropInfo = <table> {
 }
 SetBackdrop = <function> defined @SharedXML/Backdrop.lua:329
 TokenFrame = TokenFrame {
 }
 PixelSnapDisabled = true
 SetBorder = <function> defined @SharedXML/PortraitFrame.lua:62
 layoutType = "PortraitFrameTemplate"
 SetPortraitAtlasRaw = <function> defined @SharedXML/PortraitFrame.lua:83
 SetTitleMaxLinesAndHeight = <function> defined @SharedXML/PortraitFrame.lua:49
 HasBackdropInfo = <function> defined @SharedXML/Backdrop.lua:278
 SetPortraitToBag = <function> defined @SharedXML/PortraitFrame.lua:75
 PortraitContainer = Frame {
 }
 SetPortraitToUnit = <function> defined @SharedXML/PortraitFrame.lua:71
 OnBackdropSizeChanged = <function> defined @SharedXML/Backdrop.lua:182
 SetPoint = <function> defined =[C]:-1
 DefaultPortraitTextureSizeAndOffset = <function> defined @SharedXML/PortraitFrame.lua:101
 GetPortrait = <function> defined @SharedXML/PortraitFrame.lua:58
 SetScale = <function> defined =[C]:-1
 NineSlice = Frame {
 }
 TopTileStreaks = Texture {
 }
 InsetRight = CharacterFrameInsetRight {
 }
 SetupTextureCoordinates = <function> defined @SharedXML/Backdrop.lua:214
 SetPortraitToAsset = <function> defined @SharedXML/PortraitFrame.lua:67
 SetPortraitTextureSizeAndOffset = <function> defined @SharedXML/PortraitFrame.lua:95
 GetEdgeSize = <function> defined @SharedXML/Backdrop.lua:188
 CloseButton = CharacterFrameCloseButton {
 }
 numTabs = 3
 Bg = CharacterFrameBg {
 }
 SetupPieceVisuals = <function> defined @SharedXML/Backdrop.lua:246
 SetBackdropColor = <function> defined @SharedXML/Backdrop.lua:399
 selectedTab = 1
 SetTitleFormatted = <function> defined @SharedXML/PortraitFrame.lua:40
}
(*temporary) = "TOPLEFT"
(*temporary) = "UIParent"
(*temporary) = "TOPLEFT"
(*temporary) = 16
(*temporary) = -116
```

**Comment by jaredbwasserman** [comment]
This is fixed in Drift v2.2.1.
